Makefile

```
@@ -0,0 +1,71 @@
+##
+## ----------------
+## General
+## ----------------
+##
+
+COMPOSE_EXEC_PHP=docker-compose exec api
+
+all: help
+
+help: ## Display this message
+	@grep -E '(^[a-zA-Z0-9_\-\.]+:.*?##.*$$)|(^##)' Makefile | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
+
+install: build start ## Bootstrap project
+	make api_composer CMD="install"
+
+start: ## Start
+	make -j 1 start-api
+
+start-api: ## Start API container
+	docker-compose up -d
+	docker-compose start
+	${COMPOSE_EXEC_PHP} symfony server:start -d
```

J'ai un warning en faisant `make start` après un `make rm` qui me suggère d'ajouter cette option

 [WARNING] run "symfony server:ca:install" first if you want to run the web server with TLS support, or use "--p12" or  
  "--no-tls" to avoid this warning                                                                                      
                                    

```suggestion
	${COMPOSE_EXEC_PHP} symfony server:start -d --no-tls
```

---

Corrigé :heavy_check_mark: